### PR TITLE
[IMP]add module payment_alipay for v8.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: payment_alipay/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline payment_alipay/__openerp__.py:1]: short headers

File: payment_alipay/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Alipay Payment Acquirer',
+    'category': 'Hidden',
```
Comment (elicoidal) [inline payment_alipay/__openerp__.py:6]: Why hidden? Is is standard?

File: payment_alipay/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    'summary': 'Payment Acquirer: Alipay Implementation',
+    'version': '8.0.2.0.1',
+    'description':
```
Comment (elicoidal) [inline payment_alipay/__openerp__.py]: move to README

File: payment_alipay/data/alipay.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+<openerp>
+    <data noupdate="0">
+
+        <record id="payment_acquirer_alipay" model="payment.acquirer">
+            <field name="name">Alipay</field>
+            <field name="provider">alipay</field>
+            <field name="company_id" ref="base.main_company"/>
+            <field name="view_template_id" ref="alipay_acquirer_button"/> 
+            <field name="environment">test</field>
+            <field name="pre_msg"><![CDATA[
```
Comment (elicoidal) [inline payment_alipay/data/alipay.xml:11]: Is it demo data?
If this data is specific to Elico, please remove.
Better have parameters in properties/setup

File: payment_alipay/tests/test_main.py [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+        # create an payment acquirer for testing
+        self.payment_acquirer = self.env['payment.acquirer'].create(
+            {'name': 'alipay', 'provider': 'alipay',
+                'website_published': True,
+                'alipay_pid': 000000,
+                'alipay_seller_email': 'luke.zheng@elico-corp.com',
+                'view_template_id': 1,
+                'alipay_key': 1,
+                'service': 'create_direct_pay_by_user'})
+        self.return_data = {
```
Comment (elicoidal) [inline payment_alipay/tests/test_main.py:30]: This data is for our company. Not sure this is a good idea.

File: payment_alipay/tests/test_sale_order.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        # # create an empty sale order for testing
+        self.sale_order = self.env['sale.order'].create(
+            {'partner_id': 1}
+        )
+        # # create an payment acquirer for testing
+        self.payment_acquirer = self.env['payment.acquirer'].create(
+            {'name': 'alipay', 'provider': 'alipay',
+                'website_published': True,
+                'alipay_pid': 000000,
+                'alipay_seller_email': 'luke.zheng@elico-corp.com',
```
Comment (elicoidal) [inline payment_alipay/tests/test_sale_order.py:20]: isnot there any demo account usable for this cases?

File: payment_alipay/zh_CN(1).po [hunk 7]
```diff
@@ -0,0 +1,38 @@
+# Translation of Odoo Server.
```
Comment (elicoidal) [inline payment_alipay/zh_CN(1).po]: name of the file

**Comment by elicoidal** [comment]
remove hidden files (.DS..._)

**Comment by Rona111** [comment]
@elicoidal  this branch is in progress,I can't add a labels,please don't comment now.Tks.

File: payment_alipay/__openerp__.py (hunk 2 above)
Comment (Rona111) [inline payment_alipay/__openerp__.py:6]: @elicoidal  this is the same as  the official's .

**Comment by victor-ditay** [comment]
@elicoidal @seb-elico This one should be marked with label To be reviewed? Last commit tag is WIP. 

@Rona111 how much do you estimate is necessary to work to finish?

**Comment by seb-elico** [comment]
@victormartinelicocorp I think @noahzaozao already migrated this module(s) to OCA. Right?

**Comment by victor-ditay** [comment]
> @victormartinelicocorp I think @noahzaozao already migrated this module(s) to OCA. Right?

@noahzaozao ?

**Comment by elicoidal** [comment]
@victormartinelicocorp Yes and it should be updated with latest @noahzaozao version

**Comment by elicoidal** [comment]
I would close this one

**Comment by noahzaozao** [comment]
@elicoidal @seb-elico https://github.com/OCA/l10n-china/pull/21 Please use this PR , it's latest alipay module.

**Comment by seb-elico** [comment]
@noahzaozao so you confirm we can close this one, right?

**Pull request closed without merging**